src/pages/Comments/hooks/usePostDetail.ts

```diff
@@ -0,0 +1,44 @@
+import { gql } from "graphql-request";
+import { useQuery } from "react-query";
+import { GetPostDetailQuery, GetPostDetailQueryVariables } from "src/generated/graphql";
+
+import useQueryClient from "src/hooks/useQueryClient";
+
+const usePostDetail = (postId: string) => {
+    const queryClient = useQueryClient();
+
+    const result = useQuery<GetPostDetailQuery, { message: string }>(
+        ["GetPostDetail"],

```

Comment:
Are you sure this is the only key?

Reply:
Okay I will change it to "GetPostDetailPage" to be more clearly